__tests__/helpers/Signatures.rei

```diff
@@ -1,183 +0,0 @@
-module Mappable: {
```

Comment:
I didn't mean for this to be removed entirely, just the`.rei`. Effectively renaming the `.rei` to `.re`. But they don't both need to exist.

Reply:
Did mean to take it out. Was operating on fresh clones to see if I could formatting to work and didnt add back everything. Sorry about that.